### ISSUE TITLE
Slide Duration configuration does not work

### ISSUE THREAD

**Issue by robinovsky** [FlowingCode/CarouselAddon#38]
The property slideDuration seems to not be set properly. Whatever value is set, it always default to 5 seconds (default in the @xpertsea/paper-slider javascript)

Probable cause:
At Line 98 in Carousel.java,` this.getElement().setProperty` is used
```
 public void setSlideDuration(int slideDuration) {
    this.getElement().setProperty(SLIDE_DURATION, slideDuration);
  }
```

Shouldn't ` this.getElement().setAttribute` be used instead as with the other parameters ?

**Comment by javier-godoy** [comment]
Yes, I think it makes sense. Thanks for calling this into our attention.

**Comment by javier-godoy** [comment]
The attribute `slide-duration` is mapped to the property `slideDuration`. We were using the wrong name with setProperty.

https://github.com/Link2Twenty/l2t-paper-slider/blob/5f75992afba75bce725e7d6772e3b3ec25ad00ee/l2t-paper-slider.html#L272-L285

**Comment by javier-godoy** [comment]
Hello. The change has been merged into 2.1.3-SNAPSHOT, which is available from https://maven.flowingcode.com/snapshots/
